tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+
+## Usage##
+
+We are ready to interact with Redshift using the spark-redshift library. The skeleton of the program we will be using is shown in Listing 1. The entire `SparkRedshiftTutorial.scala` program can be accessed from [here](SparkRedshiftTutorial.scala). You can also use the Spark REPL to run the lines listed in the program below. 
+
+```scala
+package com.databricks.spark.redshift.tutorial
+import org.apache.spark.{SparkConf,SparkContext}
+import org.apache.spark.sql.SaveMode
+import java.io.File
+import org.apache.spark.sql.types.{StructType,StructField,StringType,IntegerType,LongType,DecimalType}
+
+
+object SparkRedshiftTutorial {
+  /*
+   * For Windows Users only
+   * 1. Download contents from link https://github.com/srccodes/hadoop-common-2.2.0-bin/archive/master.zip
+   * 2. Unzip the file in step 1 into your %HADOOP_HOME%/bin. In the example below  %HADOOP_HOME%=C:/MySoftware/hdphome
+   * 3. After step 2 %HADOOP_HOME/bin should contain winutils.exe
+   * 
+  */
+  if(System.getProperty("os.name").startsWith("Windows")){
+    val hdphome:File = new File("C:/MySoftware/hdphome");
+    System.setProperty("hadoop.home.dir",hdphome.getAbsolutePath().toString())    
+  }
+  
+  def main(args:Array[String]):Unit = {
+    val awsAccessKeyId = args(0) 
+    val awsSecretAccessKey = args(1)    
+    val redshiftDBName = args(2)
+    val redshiftUserId= args(3)
+    val redshiftPassword= args(4)
+    /**
+      * Sample URL-swredshift.czac2vcs84ci.us-east-1.redshift.amazonaws.com:5439
+      */
+    val redshifturl = args(5) 
+    val jdbcURL = s"jdbc:redshift://$redshifturl/$redshiftDBName?user=$redshiftUserId&password=$redshiftPassword"
+
+    val sc = new SparkContext(new SparkConf().setAppName("SparkSQL").setMaster("local"))
+
+	/*Configure SparkContext to communicate with AWS*/
+	val tempS3Dir = "s3n://redshift-spark/temp/"
+    sc.hadoopConfiguration.set("fs.s3n.awsAccessKeyId", awsAccessKeyId)
+    sc.hadoopConfiguration.set("fs.s3n.awsSecretAccessKey", awsSecretAccessKey)
+
+	/*Create the SQL Context*/    
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+  }
+} 
+```
+Listing 1 : SparkRedshiftExamples.scala skeleton
+
+We need the following set of user provided parameters to communicate with AWS in general and Redshift in particular -
+
+- **AWS Access Key and AWS Secret Access Key** - This key pair will be used to communicate with AWS services. This information is passed by the AWS Client libraries in every interaction with AWS.
+- **Redshift Database Name** - When you provision the Redshift service you have to provide a name of for your database. This is similar to a schema in Oracle. The name of my Redshift database was sparkredshift
```

Comment:
I'd probably wrap sparkredshift in markdown backticks, similar to the O'Reilly etc. style for drawing attention to string literals in their prose.